### ISSUE TITLE
config file not found - Shutting down...

### ISSUE THREAD

**Issue by jacobmix** [Pizzabelly/EasyRP#72]
Sometimes when i edit the config a lot it just starts saying it can't find it, and it's shutting down.
It never shuts down tho, and the config file is clearly still there.
Maybe make it check a few times before it tries to shut down?

**Comment by LavaJockey** [comment]
Are you using a batch script to use it?

**Comment by jacobmix** [comment]
Yeah. I use a bat script that watches a program, and changes the config.ini when the program has changes.
